### ISSUE TITLE
Notifier is not calling .waitFor before .exitCode() check, causing .exitCode() to always fail

### ISSUE THREAD

**Issue by deathbeam** [runelite/runelite#5956]
```
2018-10-12 10:21:04 [pool-2-thread-1] WARN  n.r.c.util.RunnableExceptionLogger - Uncaught exception in runnable net.runelite.client.Notifier$$Lambda$418/1163862728@7733819c
java.lang.IllegalThreadStateException: process hasn't exited
        at java.lang.UNIXProcess.exitValue(UNIXProcess.java:423)
        at net.runelite.client.Notifier.lambda$null$0(Notifier.java:212)
        at java.util.Optional.map(Optional.java:215)
        at net.runelite.client.Notifier.lambda$sendLinuxNotification$1(Notifier.java:212)
        at net.runelite.client.util.RunnableExceptionLogger.run(RunnableExceptionLogger.java:41)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
```

**Comment by marek-urban** [comment]
Greetings.
I would like to be involved in fixing this issue.

**Comment by Adam-** [comment]
Fixed in 431713b9a4e90bfb4a2424b702ed7a09051ddc15
